src/PostalRegistry.Api.Legacy/PostalInformation/Query/PostalInformationSyndicationQuery.cs

```diff
@@ -26,20 +26,13 @@ public class PostalInformationSyndicationQueryResult
         public Instant LastChangedOn { get; }
         public PostalInformationStatus? Status { get; }
         public IEnumerable<PostalName> PostalNames { get; }
-        public string MunicipalityNisCode { get; }
-        public Organisation? Organisation { get; }
-        public string Reason { get; }
-        public string EventDataAsXml { get; }
 
         public PostalInformationSyndicationQueryResult(
             string postalCode,
             long position,
             string changeType,
             Instant recordCreatedAt,
-            Instant lastChangedOn,
-            string municipalityNisCode,
-            Organisation? organisation,
-            string reason)
+            Instant lastChangedOn)
```

Comment:
what is happening in this file? Why are things being deleted/commented in our sync service?